shared/clinical-code-sets/conditions/uti/2/uti.ctv3.txt

```diff
@@ -0,0 +1,130 @@
+A160.	Renal tuberculosis
+A1600	Tuberculous nephropathy
+A1601	Tuberculous pyelitis
+A1602	Tuberculous pyelonephritis
+A160z	Tuberculosis of kidney NOS
+A162.	Tuberculosis of ureter
+A163.	Tuberculosis of other urinary organs
+A32y3	Diphtheritic cystitis
+A8441	Plasmodium malariae malaria with nephropathy
+A954.	Syphilis of kidney
+A9811	Acute gonococcal cystitis
+A9831	Chronic gonococcal cystitis
+A994.	Nonspecific urethritis
+AC2z0	Hydatid cyst of kidney
+AD102	Trichomonal urethritis
+K01x2	Nephrotic syndrome in malaria
+K0y0.	Late syphilis of kidney
+K10..	Infection of kidney
+K100.	Chronic pyelonephritis
+K1001	Chronic pyelonephritis with medullary necrosis
+K1003	Chronic pyonephrosis
+K1004	Non-obstructive reflux-associated chronic pyelonephritis
+K1006	Calculous pyelonephritis
+K100z	Chronic pyelonephritis NOS
+K101.	Acute pyelonephritis
```

Comment:
the README says this code set excludes pyelonephritis but there are a few codes here

Reply:
Code removed and prevalence updated.